hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsSchedulerModule.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.mesos.hdfs.scheduler;
```

Comment:
Lets talk about this... this adds a lot of value in the ability to test with mocks.   I'm not sure how it is overcomplicated... it is well known and understood.    It is a necessary step when using guice with interfaces.